Source/ui/UI.js

```diff
@@ -92,6 +93,48 @@ export function getInputFromUser(messageText, options, callback) {
       )
       dialog.window().setInitialFirstResponder(accessory)
       break
+    case INPUT_TYPE.textarea:
```

Comment:
Do you think it's necessary to add another input type? I was thinking of just having it as having the `numberOfLines` option default to 1  and if it's different than 1, then change the text input to be a textarea

Reply:
Ah fair point. That is probably easiest since the other input type is `string` and not `textfield`. Though both on the web and on macOS those are pretty explicitly two different objects.

Reply:
Moved it over into the `string` type. Seems like the switch statement is getting pretty big now!

Reply:
I think that's fine, we can refactor into smaller functions when we need it